code/dynamic_programming/longest_bitonic_sequence/longest_bitonic_sequence.py

```diff
@@ -0,0 +1,49 @@
+# Part of Cosmos by OpenGenus Foundation
+
+import copy
+# you can use any input format
+# format the input such that the array of numbers looks like below
+numbers = [1, 11, 2, 10, 4, 5, 2, 1]
+
+# longest increasing subsequence
+lis = []
+# base case
+lis.append([numbers[0]])
+
+# start filling using dp - forwards
+
+for i in xrange(1, len(numbers)):
+    lis.append([])
+    for j in xrange(i):
+        if numbers[j] < numbers[i] and len(lis[j]) > len(lis[i]):
+            lis[i] = copy.copy(lis[j])
+    lis[i].append(numbers[i])
+
+# longest decreasing subsequence
+lds = []
+
+# to make an n sized list
+for _ in xrange(len(numbers)):
+    lds.append([])
```

Comment:
Don't you think it is simpler to use list comprehension here? 
Like `lds = [ [] for _ in range(len(numbers)) ]`